File: Leet-Codes/largerNumberArray.js
var numbers = parseInt(prompt("digite quantos  numeros quer escrever: "));
numbersList = [];


for(terms = 0; terms < numbers; terms++){
    var n = parseFloat(prompt("digite seus numeros: "));
    numbersList.push(n);
}

var larger = Math.max(...numbersList);
alert("O maior numero dado eh: " + larger);

//Por esse código num arquivo html padrão dentro de uma tag script ou no src="esse arquivo"

File: Leet-Codes/largerNumberArray.py
listNumbers = []
i = int(input("Digite quantos valores deseja que sejam adicionados na sua lista: "))

for terms in range(i):
    n = input("Digite o valor a ser adicionado na sua lista: ")
    listNumbers.append(n)

larger = max(listNumbers)
print("O maior numero da sua lista eh " + larger)